### PR TITLE
Feat/pn 11245

### PR DESCRIPTION
**Pull request by angelominisci**
## Short description

Include a summary of the changes.

## List of changes proposed in this pull request

- Feature A
- Feature B

## How to test

Describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

- [ ] Does this PR require a manual test in AWS CodeBuild?

### CODE REVIEW

File: src/test/resources/feature/2-mittente/Sequence/invioNotificaConSequence_fail_AR.feature [hunk 1]
```diff
@@ -44,4 +34,7 @@ Feature: invio notifica con sequence
     And Si controlla lo stato timeline in dettaglio notifica
       | xpathStato   | //p[contains(text(),"La raccomandata A/R") and contains(text(),"non è stata consegnata")] |
       | vediDettagli | false                                                                                     |
-    And Logout da portale mittente
\ No newline at end of file
+    And Si verifica la cliccabilità della ricevuta di postalizzazione
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/Sequence/invioNotificaConSequence_fail_AR.feature]: delete extra /n

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 2]
```diff
@@ -1735,6 +1737,18 @@ public void siControllaLoStatoTimelineInDettaglioNotifica(Map<String, String> da
         dettaglioNotificaMittenteSection.checkStatoTimeline(idStato);
     }
 
+    @And("Si verifica la cliccabilità della ricevuta di postalizzazione")
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: use "Si verifica che la ricevuta di postalizzazione sia cliccabile"

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 3]
```diff
@@ -1735,6 +1737,18 @@ public void siControllaLoStatoTimelineInDettaglioNotifica(Map<String, String> da
...
+    public void siVerificaLaCliccabilitaDellaRicevutaDiPostalizzazione(Map<String, String> datiDettaglioNotifica) {
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:1741]: I would not have created a new method but i would inserted another parameter in Si controlla lo stato timeline in dettaglio notifica which check the xpath you have passed is clickable

File: src/test/resources/feature/2-mittente/Sequence/invioNotificaConSequence_fail_AR.feature (hunk 1 above)
Comment (angelominisci) [reply src/test/resources/feature/2-mittente/Sequence/invioNotificaConSequence_fail_AR.feature]: sistemato

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 3 above)
Comment (angelominisci) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java:1741]: sistemato

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 2 above)
Comment (angelominisci) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: sistemato

**Review by leleOFA — APPROVED** [review]

**Pull request merged**